ipfs/src/route.rs

```diff
@@ -119,7 +119,8 @@ mod tests {
 	use ethcore::client::TestBlockChainClient;
 
 	fn get_mocked_handler() -> IpfsHandler {
-		IpfsHandler::new(None, None, Arc::new(TestBlockChainClient::new()))
+		//IpfsHandler::new(None, None, Arc::new(TestBlockChainClient::new()))
```

Comment:
Agreed, I should have removed that line.

Yes, it's not compiling, and those tests were not run by our CI. Now this code is being compiled by CI, but the tests are excluded (using `--exclude`) flag.